starknet_py/tests/e2e/fixtures/proxy.py

```diff
@@ -70,6 +70,21 @@ async def deploy_proxy_to_contract_custom(
     )
 
 
+@pytest_asyncio.fixture(
+    params=[("oz_proxy_exposed_compiled.json", "map_compiled.json")]
+)
+async def deploy_proxy_to_contract_exposed(
```

Comment:
Maybe a better name would be nice? We could rename `deploy_proxy_to_contract` function to something else and name this fixture `deploy_proxy_to_contract`.

Reply:
There already is a function `deploy_proxy_to_contract`.
Solving this would require indirect parametrization + it is used twice sometimes in one test and I don't know if it's possible to even have 2 variables parametrized by the same fixture in one test.

Maybe we could just leave it

Reply:
It's not a big problem. It's just a bit weird name.